### PR TITLE
rubocop configurations from bundler/bundler

### PR DESCRIPTION
**Pull request by fotanus**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,7 +1,95 @@
-inherit_from: .rubocop_todo.yml
+inherit_from:
+  - .rubocop_todo.yml
 
 AllCops:
   Exclude:
-    - bin/**/*
-    - vendor/**/*
```
Comment (segiddins) [inline .rubocop.yml]: Need to keep these

**Comment by segiddins** [comment]
Please include the Bundle rubocop config via `inherit_from `?

**Comment by fotanus** [comment]
@segiddins you mean I should copy the .rubocop_todo.yml as well?

**Comment by segiddins** [comment]
No, I want you to have a `.rubocop-bundler.yml` file that has the contents of the Bundler one, which then gets included in the `.rubocop.yml`

**Comment by fotanus** [comment]
This way? We still have this on rubocop-bundler.yml: 

```
inherit_from:
  - .rubocop_todo.yml
```

Should I remove?


Also, running on my machine yelds this:
```
Warning: unrecognized cop Lint/NestedMethodDefinition found in /Users/fotanus/code/bundler-api/.rubocop_todo.yml
Warning: unrecognized cop Lint/NestedMethodDefinition found in /Users/fotanus/code/bundler-api/.rubocop-bundler.yml
Warning: unrecognized cop Lint/NestedMethodDefinition found in /Users/fotanus/code/bundler-api/.rubocop.yml
```
which do not seems to happens when I run on bundler/bundler.

**Comment by segiddins** [comment]
@fotanus try `bundle update rubocop`.

**Comment by fotanus** [comment]
:+1: Like a charm, thanks!

**Comment by fotanus** [comment]
The only issue left is:
```
inherit_from:
  - .rubocop_todo.yml
```
Should we remove this?

**Comment by segiddins** [comment]
@fotanus if it's unused?

**Comment by fotanus** [comment]
Removing I got this: `46 files inspected, 992 offenses detected`

With it I got only `46 files inspected, no offenses detected`

So should I fix all ofenses or include the todo? :-)

**Comment by fotanus** [comment]
What do you think @indirect ?

**Comment by indirect** [comment]
for now, we need the todos :)

**Comment by fotanus** [comment]
Ok, so this should be good to go!

**Pull request merged**